packages/dx-react-chart/demos/bar/fullstacked-bar.md

```diff
@@ -0,0 +1,5 @@
+# React Chart Fullstacked Bar
+
+The demo shows how to create Fullstacked Bar Series using the [Stack](../../docs/reference/stack.md) plugin.
```

Comment:
The demo shows how to create Fullstacked Bar Series using the [Stack](../../docs/reference/stack.md) plugin.
->
The demo shows how to use the [Stack](../../docs/reference/stack.md) plugin to create Fullstacked Bar Series.